### ISSUE TITLE
Many articles are already removed or deleted completely

### ISSUE THREAD

**Issue by pold500** [graninas/cpp_functional_programming#2]
I suggest we save them into html format on google drive or just here

**Comment by graninas** [comment]
This might be a good idea, thanks!

I'll see what I can do.

**Comment by pold500** [comment]
What about a book on functional design? I still wait for a new chapters.

**Comment by graninas** [comment]
@pold500 Sad story, I'm still procrastinating in this direction.

Currently I'm preparing a Patreon / crowdfunding campaign to be able to get some benefit from the book. I want to start this campaign on the 4th of March. However I don't think I can start working on the book at March because I need to prepare two talks for April and May.

Yes, I know this sounds sucks. :(

**Comment by graninas** [comment]
@pold500 FYI: I started a campaign finally.

**Comment by pold500** [comment]
@graninas yep, I've seen your post on reddit /fp, already joined the tier. Don't procrastinate :)

**Comment by graninas** [comment]
@pold500 That's a good advice, thanks!

**Comment by pold500** [comment]
@graninas do you have telegram?

**Comment by graninas** [comment]
@pold500 Yes, it's the same: [@graninas](https://web.telegram.org/#/im?p=@graninas)

**Comment by pold500** [comment]
So, we need to do something actually :)
Maybe we can save them in doc format to google drive?
